### ISSUE TITLE
Project license?

### ISSUE THREAD

**Issue by Draconizations** [estroBiologist/pluralchum#9]
TL;DR. Can this project have a license lmao.

I modified pluralchum for personal purposes. Mostly just making the proxy name and tag entirely the same color, so that server settings remain preserved (as it keeps the original webhook's name). And it works nicely!

I have a couple people interested in this edited version, since the lack of server setting preservation is an issue for some of them. I just don't know if I'm allowed to spread around edited versions of this plugin. A license would be nice!

(Also if you're interested I *probably* could PR a setting that does the exact same thing: removing the different colored system tags so it keeps servernames and tags. Just let me know if that's a thing you'd like to see!)

**Comment by estroBiologist** [comment]
Hey! Sorry for the late response, apparently GitHub didn't deem it necessary to notify me about this issue... I'll go ahead and add the plain ol' MIT license to the repo - didn't even realize I forgot to add one in the first place, tbh. Sorry about that!

I'm absolutely open to PRs, although I can't promise when or if I'll have the time to publish a new proper release of PC. Still, I can totally understand the need for such a setting, so if you're so inclined I'd say go for it! Thanks 😄
